source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
+You may provide a filter of your own choosing by adding a zcl json string to the "filter" parameter
+The zcl is supplied in the same format as regular dataset filters.  Here is an example url with filter on a
+categorical variable: 
+
+`/datasets/{id}/export/csv/?filter={"function": "==", "args": [{"variable": "000000"}, {"value": 1}]}`
+
+You can provide a selection of variables with the "where" parameter.  Here is how you would limit the export to
+a single variable:
+
+`/datasets/{id}/export/spss/?where={"function": "identify", "args": [{"id": ["000000"]}]}`
 
```

Comment:
I don't know what "all zcl functions" entails here. I've never seen or worked with an "identify" function in the HTTP API, nor have I seen an element of "args" that is `{"id": something}`. So I need more here if I'm going to be able to use this API.